modules/aggregations/src/test/java/org/elasticsearch/aggregations/pipeline/CumulativeSumTests.java

```diff
@@ -6,11 +6,12 @@
  * Side Public License, v 1.
  */
 
-package org.elasticsearch.search.aggregations.pipeline;
+package org.elasticsearch.aggregations.pipeline;
```

Comment:
The pipeline tests moved in this PR use `PipelineAggregationHelperTests`, which randomly use the auto date histogram aggregator. I could have temporarily removed the usage of this aggregator, but it felt better to already move these pipeline aggregator tests to the aggregation module, since this is where the pipeline aggregators that these tests are testing will reside after these have been moved.